kserve-modelmesh.yaml

```diff
@@ -30,23 +31,27 @@ pipeline:
       export JAVA_HOME=/usr/lib/jvm/java-17-openjdk
       mvn -B package -Dfile.encoding=UTF8 -DskipTests=true --file pom.xml
       mkdir -p ${{targets.destdir}}/opt/kserve/mmesh
-      mv /home/build/target/dockerhome ${{targets.destdir}}/opt/kserve/mmesh/
-      chmod -R 775 ${{targets.destdir}}/opt/kserve/mmesh/dockerhome
+      mv /home/build/target/dockerhome/* ${{targets.destdir}}/opt/kserve/mmesh/
```

Comment:
If this works independently, it might be worth splitting into a separate package - something like `kserve-modelmesh-docker` with `bash` removed from runtime deps above and added to the subpackage?

Reply:
I'm not sure I fully follow why we need the subpackage

Reply:
> If this works independently, it might be worth splitting into a separate package

I think this looks like something that's core to this package so we can keep it here probably. It's a group of jars and few scripts and I'm not sure if we can place it at some other location. 

![Screenshot 2024-10-25 at 1 17 24 AM](https://github.com/user-attachments/assets/97a7fe3a-34ad-4363-b5f5-b1be18296421)

please correct if that's not the case.

Reply:
We don't need one. The idea was to keep Docker bits separate from the framework so that it could be installed independently without adding things that would only be useful in Docker